packages/location-management/src/components/LocationUnitGroupAddEdit/Form.tsx

```diff
@@ -0,0 +1,169 @@
+import * as Yup from 'yup';
+import React, { useEffect, useState } from 'react';
+import { SubmitButton, Form as AntForm, Input, Radio } from 'formik-antd';
+import { Button } from 'antd';
+import { history } from '@onaio/connected-reducer-registry';
+import { OpenSRPService } from '@opensrp/server-service';
+import { getAccessToken } from '@onaio/session-reducer';
+import { Formik } from 'formik';
+import { useSelector } from 'react-redux';
+import { LOCATION_UNIT_GROUP_ALL, LOCATION_UNIT_GROUP_GET } from '../../constants';
+import { API_BASE_URL } from '../../configs/env';
```

Comment:
Let's retrieve the base URL from props

Reply:
@haseemVD have created a separate issue to deal with it .. it will be solved separately

Reply:
Created separate [issue](https://github.com/OpenSRP/web/issues/256) for this